native_client/coqui-stt.h

```diff
@@ -326,6 +326,20 @@ Metadata* STT_IntermediateDecodeWithMetadata(const StreamingState* aSctx,
  * @note This method will free the state pointer (@p aSctx).
  */
 STT_EXPORT
+char* STT_IntermediateDecodeExpensive(StreamingState* aSctx);
+
+/**
+ * @brief Compute the intermediate decoding of an ongoing streaming inference,
```

Comment:
This `@brief` will go in the docs, so can we expand it more, like you did for the python bindings?

Reply:
Also, explain in a few words why this is "expensive"

Reply:
Roger that